NEWS.txt

```diff
@@ -38,6 +38,14 @@ using the provided 'sstableupgrade' tool.
 
 New features
 ------------
+    - The data of the system keyspaces using a local strategy (at the exception of the system.batches,
+      system.paxos, system.compaction_history, system.prepared_statements and system.repair tables)
+      is now stored by default in the first data directory. This approach will allow the server
```

Comment:
Perhaps `is now stored by default in the first data directory, instead of being distributed among all the data directories.`, to be even more clear?